pootle/apps/accounts/models.py

```diff
@@ -60,6 +60,10 @@ class User(AbstractBaseUser):
                            _('Enter a valid username.'),
                            'invalid')
         ],
+        # Remove when fixed in Django: https://code.djangoproject.com/ticket/27898#ticket
```

Comment:
Please try use something like `FIXME` as tools can flag that in future.  Apart from the fact that we don't need this comment at all.